### ISSUE TITLE
Marvels Guardian of the Galaxy (Black boxes and flickering)

### ISSUE THREAD

**Issue by ukbeast** [HansKristian-Work/vkd3d-proton#1835]
Please describe your issue as accurately as possible. Include screenshots or videos if relevant

### Software information
Marvels Guardian of the Galaxy, via heoric launcher
![Screenshot_20240116_115003](https://github.com/HansKristian-Work/vkd3d-proton/assets/6401168/6344a043-71d0-4571-8110-04b62c6c6ccb)
![Screenshot_20240116_115018](https://github.com/HansKristian-Work/vkd3d-proton/assets/6401168/dd7b4474-b19b-44ad-93da-9c75c181f075)
.

### System information
- GPU: Radeon RX 6600
- Driver: radv 23.3.3
- Wine version: wine-9.0-rc3 (Staging)
- VKD3D-Proton version: Version 2.11.1

### Log files
[vkd3d-proton.log](https://github.com/HansKristian-Work/vkd3d-proton/files/13949529/vkd3d-proton.log)

**Comment by Blisto91** [comment]
Hi there. 

I am not reproducing this issue so far using a RX 6800 with the same driver and vkd3d-proton version. Running it through Steam Proton.
Any specific settings that makes it appear?

**Comment by ukbeast** [comment]
It's an Epic version, I am playing this through heroic launcher and I am using the staging version of wine.

**Comment by Blisto91** [comment]
I meant ingame settings. If it's either a driver or vkd3d-proton issue then the store shouldn't matter.

**Comment by Riesi** [comment]
works here with the Epic version on an GT980 at just Ultra preset:
![image](https://github.com/HansKristian-Work/vkd3d-proton/assets/27691485/126f43d6-f3a8-467b-bf89-b21930f3e2f5)

**Comment by shelterx** [comment]
Yes, GOTG works with Nvidia but it got some texture flickering here and there. I always get some flickering in the benchmark at the end (the sky behind the characters).
RTX 4070 - RT On, ultra settings.

**Comment by DomiStyle** [comment]
Not sure if it's the exactly same issue but I'm also getting flickering on my 6900 XT running through gamescope with HDR enabled. Will try later without gamescope and HDR to see if the issue lies there.

Tried with Proton 9 (Beta) and Proton Experimental on the Steam version.

Looks more like some geometry is going nuts though:

![Screenshot_20240304_162920](https://github.com/HansKristian-Work/vkd3d-proton/assets/975131/3104f6c3-2e96-4b2a-949e-776f24b71347)

For comparison, here's the frame after that:

![Screenshot_20240304_162928](https://github.com/HansKristian-Work/vkd3d-proton/assets/975131/702d6854-9734-452e-ab4e-2a9aabeae03f)
